semanticdb/semanticdb/semanticdb.proto

```diff
@@ -2,6 +2,12 @@ syntax = "proto3";
 
 package scala.meta.internal.semanticdb;
 
+import "scalapb/scalapb.proto";
+
+option (scalapb.options) = {
+  preserve_unknown_fields: false
```

Comment:
Is this a backwards compatible change?

Reply:
Yes.
This option suppresses `unknownFieldsSet` parameter to be added.
If not added, lots of files needs modification and also needs copying older sources for Scala 2.11.
I think such modifications is harmful in terms of maintainability.